webrtc.html

```diff
@@ -4978,7 +4979,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             "idlMemberType"><a>DOMString</a></span></dt>
             <dd>
               <p>The codec MIME type. Valid types are listed in
-              [[IANA-RTP-2]].</p>
+              [[IANA-RTP-2]]. RTP retransmission MIME types are not allowed.</p>
```

Comment:
I think you need to change this to refer to this particular type of RTP retransmision (prob just ref the RFC) as other types might still be in there.